### PR TITLE
Add test for T1170 that launches local notepad via VBScript called by…

### PR DESCRIPTION
**Pull request by azeemnow**
… Mshta

**Details:**

**Testing:**

**Associated Issues:**

### CODE REVIEW

File: atomics/T1170/T1170.yaml [hunk 1]
```diff
@@ -18,3 +18,132 @@ atomic_tests:
     command: |
       mshta.exe javascript:a=(GetObject('script:#{file_url}')).Exec();close();
       
+
+- name: Mshta calls a local vbs file to launch Notepad.exe
```
Comment (keithmccammon) [inline atomics/T1170/T1170.yaml]:
```suggestion
- name: Mshta calls a local VBScript file to launch notepad.exe
```

File: atomics/T1170/T1170.yaml [hunk 2]
```diff
@@ -18,3 +18,132 @@ atomic_tests:
...
+  description: Tests execution of a local program by a VBScript file called by Mshta
+
+  supported_platforms:
+    - windows
+
+  input_arguments:
+    local_file: 
+      description: create a local vbs file
```
Comment (keithmccammon) [inline atomics/T1170/T1170.yaml]:
```suggestion
      description: Create a local VBScript file
```

File: atomics/T1170/T1170.yaml [hunk 3]
```diff
@@ -18,3 +18,132 @@ atomic_tests:
...
+      type: strings
+      default: Set objShell = WScript.CreateObject("WScript.Shell") objShell.Run ("%windir%\notepad.exe ")
```
Comment (keithmccammon) [inline atomics/T1170/T1170.yaml]: This should be a default local_file path and description. Also note that local_file_path below won't match this local_file input argument. These should be consistent.

File: atomics/T1170/T1170.yaml [hunk 4]
```diff
@@ -18,3 +18,132 @@ atomic_tests:
...
+    
+
+  executor:
+    name: command_prompt
+    command: |
+      mshta.exe vbscript:Execute("CreateObject(""Wscript.Shell"").Run(""{local_file_path}"")(window.close)")
+
+
+- name: TODO
```
Comment (keithmccammon) [inline atomics/T1170/T1170.yaml]: Unsure why the TODOs and blank sections exist.

File: atomics/T1170/T1170.yaml (hunk 3 above)
Comment (azeemnow) [reply atomics/T1170/T1170.yaml]: Updated to include the changes. 
Where do you I include contents of the script file?
`
image
`

**Pull request merged**